vite.config.ts

```diff
@@ -1,19 +1,22 @@
-import { defineConfig, splitVendorChunkPlugin } from 'vite';
-import { viteStaticCopy } from 'vite-plugin-static-copy';
-import dotenv from 'dotenv';
+import dotenv from "dotenv";
```

Comment:
Not sure why you changed to double quotes.

Reply:
lack of global prettier rules